### PR TITLE
legacy API code generation to match swagger files

### PR DESCRIPTION
**Pull request by pastorsx**
API regeneration using latest swagger files:
mec011
mec012
mec013
mec028

testing:
UT
system

### CODE REVIEW

File: go-apps/meep-loc-serv/api/swagger.yaml [hunk 1]
```diff
@@ -3015,28 +3015,6 @@ components:
       properties:
         problemDetails:
```
Comment (dilallkx) [inline go-apps/meep-loc-serv/api/swagger.yaml:3016]: _Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

File: go-apps/meep-rnis/api/swagger.yaml [hunk 2]
```diff
@@ -3633,28 +3633,6 @@ components:
       - 30
       - 31
```
Comment (dilallkx) [inline go-apps/meep-rnis/api/swagger.yaml:3634]: _Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

File: go-apps/meep-wais/api/swagger.yaml [hunk 3]
```diff
@@ -3031,28 +3031,6 @@ components:
           $ref: '#/components/schemas/VhtCapabilities'
       type: object
```
Comment (dilallkx) [inline go-apps/meep-wais/api/swagger.yaml:3032]: _Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

**Review by dilallkx — CHANGES_REQUESTED** [review]

File: go-apps/meep-app-enablement/api/app-support/swagger.yaml [hunk 4]
```diff
@@ -4,24 +4,20 @@ servers:
 info:
   title: AdvantEDGE MEC Application Support API
   version: 2.1.1
-  description: The ETSI MEC ISG MEC011 MEC Application Support API described using OpenAPI
+  description: "MEC Application Support API describe is AdvantEDGE's implementation of [ETSI MEC ISG MEC011 MEC Application Support API](https://www.etsi.org/deliver/etsi_gs/MEC/001_099/011/02.01.01_60/gs_MEC011v020101p.pdf)"
```
[the remark this answers is not in the document]
Comment (pastorsx) [reply go-apps/meep-app-enablement/api/app-support/swagger.yaml:7]: updated to follow rnis etc...

File: go-apps/meep-app-enablement/api/app-support/swagger.yaml [hunk 5]
```diff
@@ -4,24 +4,20 @@ servers:
...
+    url: "https://github.com/InterDigitalInc/AdvantEDGE/blob/master/LICENSE"
   contact:
-    email: cti_support@etsi.org
+    name: InterDigital AdvantEDGE Support
+    email: AdvantEDGE@InterDigital.com
 externalDocs:
   description: 'ETSI GS MEC011 Application Enablement API, V2.1.1'
   url: >-
-    https://www.etsi.org/deliver/etsi_gs/MEC/001_099/011/02.01.01_60/gs_MEC011v020101p.pdf
+    https://www.etsi.org/deliver/etsi
```
[the remark this answers is not in the document]
Comment (pastorsx) [reply go-apps/meep-app-enablement/api/app-support/swagger.yaml:17]: fixed as part of fixing headers

File: go-apps/meep-app-enablement/api/service-mgmt/swagger.yaml [hunk 6]
```diff
@@ -4,22 +4,19 @@ servers:
 info:
   title: AdvantEDGE MEC Service Management API
   version: 2.1.1
-  description: The ETSI MEC ISG MEC011 MEC Service Management API described using OpenAPI
+  description: "MEC Service Management API describe is AdvantEDGE's implementation of [ETSI MEC ISG MEC011 MEC Service Management API](https://www.etsi.org/deliver/etsi_gs/MEC/001_099/011/02.01.01_60/gs_MEC011v020101p.pdf)"
```
[the remark this answers is not in the document]
Comment (pastorsx) [reply go-apps/meep-app-enablement/api/service-mgmt/swagger.yaml:7]: same as above

File: go-apps/meep-loc-serv/api/swagger.yaml (hunk 1 above)
Comment (pastorsx) [reply go-apps/meep-loc-serv/api/swagger.yaml:3016]: part of header fix

File: go-apps/meep-rnis/api/swagger.yaml (hunk 2 above)
Comment (pastorsx) [reply go-apps/meep-rnis/api/swagger.yaml:3634]: as per above

File: go-apps/meep-wais/api/swagger.yaml (hunk 3 above)
Comment (pastorsx) [reply go-apps/meep-wais/api/swagger.yaml:3032]: as per above

**Review by dilallkx — APPROVED** [review]

**Pull request merged**